src/connector/src/sink/mysql.rs

```diff
@@ -40,8 +41,8 @@ pub struct MySQLSink {
 }
 
 impl MySQLSink {
-    pub fn new(cfg: MySQLConfig) -> Self {
-        Self { cfg }
+    pub fn new(cfg: MySQLConfig) -> Result<Self> {
+        Ok(Self { cfg })
```

Comment:
Why return Result when it's always Ok?

Reply:
it is for a unified interface of sink. I think we will do more checks on MySQL config later just like kafka